### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Geromino**
need Resistor on pin PB9 ------> I2C1_SDA internal not enough Resistor

### CODE REVIEW

**Comment by Geromino** [comment]
how add electric scheme

**Comment by Letme** [comment]
We do not need electric scheme. Just write next to the readme the remark and everyone will know what to do. Otherwise you can produce a image of scheme and add it into a folder (images?).

File: README.md [hunk 1]
```diff
@@ -27,6 +29,16 @@ IDE [ARM Keil uVision 5](https://www.keil.com/demo/eval/arm.htm)
 | pin 5: ADDR     | CN7:  pin 20: GND      |
 
 
+# Hardware connections
+| MLX90632        |stm32f4discovery        |
+| --------------- |:----------------------:|
```
Comment (Letme) [inline README.md]: the `:` here are not needed and should be removed

File: README.md [hunk 2]
```diff
@@ -27,6 +29,16 @@ IDE [ARM Keil uVision 5](https://www.keil.com/demo/eval/arm.htm)
 | pin 5: ADDR     | CN7:  pin 20: GND      |
 
 
+# Hardware connections
+| MLX90632        |stm32f4discovery        |
```
Comment (Letme) [inline README.md]: add space before STM32F4 as well as separate words so it end up: `| STM32F4 Discovery`

File: README.md [hunk 3]
```diff
@@ -27,6 +29,16 @@ IDE [ARM Keil uVision 5](https://www.keil.com/demo/eval/arm.htm)
...
+| --------------- |:----------------------:| 
+| pin 1: SDA      | p2:  pin 20: PB9      |
```
Comment (Letme) [inline README.md]: Add a remark about additional external resistor needed here.

**Review by Letme — CHANGES_REQUESTED** [review]

**Comment by Letme** [comment]
Thanks for the contribution.

**Pull request merged**